tests/queries/0_stateless/01305_duplicate_order_by_and_distinct.sql

```diff
@@ -0,0 +1,32 @@
+set enable_debug_queries = 1;
+set optimize_duplicate_order_by_and_distinct = 1;
+
+analyze SELECT DISTINCT *

```

Comment:
We need both analyze and execute it in test. Cause it's not clear would the rewritten query finish or fail.
In case of distinct we need duplicates. I.e. `select number % 2 as number from numbers(3)`